client/js/components/shared/TextContentRenderer.vue

```diff
@@ -0,0 +1,56 @@
+<script>
+import { DpLoading } from '@demos-europe/demosplan-ui'
+
+export default {
+  name: 'DpTextWrapper',
```

Comment:
The component name should match the file name.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2169/commits/98d20583e3012ef3e7021f0eb95fdea06ae459ca